packages/console/src/components/AppLoading/index.tsx

```diff
@@ -1,16 +1,19 @@
+import { useContext } from 'react';
+
 import IllustrationDark from '@/assets/images/loading-illustration-dark.svg';
 import Illustration from '@/assets/images/loading-illustration.svg';
 import { Daisy as Spinner } from '@/components/Spinner';
-import { useTheme } from '@/hooks/use-theme';
+import { AppThemeContext } from '@/contexts/AppThemeProvider';
+import { Theme } from '@/types/theme';
 
 import * as styles from './index.module.scss';
 
 const AppLoading = () => {
-  const theme = useTheme();
+  const { theme } = useContext(AppThemeContext);
```

Comment:
nit: could update the useTheme hook, wrap the 
`const { theme } = useContext(AppThemeContext);`
with in the hook. 
So we could still simple call `useTheme` to get the theme value.

Reply:
I think it redundant.

Reply:
useTheme feels more friendly than useContext, users do not need to be aware of the context every.  Anyway, it's up to you.